src/index.js

```diff
@@ -1,130 +1,282 @@
-const { REST } = require('@discordjs/rest');
-const { Client, Intents, MessageEmbed } = require('discord.js');
-const { Routes } = require('discord-api-types/v9');
-const fs = require('fs');
-const path = require('path');
-const { Sequelize } = require('sequelize');
-const { env, token } = require('../config.json');
-const reactionRoles = require('../reaction-roles.json');
-require('colors');
+import dotenv from "dotenv";
```

Comment:
switched to ES6/module style, updated to new Discord.js. introduced .env for secrets.